src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1352,16 +1359,22 @@ private void requireUnsigned(final String tag) {
     }
 
     /**
+     * @return SAMBinaryTagAndValue may be null if
+     *          1) {@link #mAttributes} is null or
+     *          2) this record does not have the requested attribute
+     */
+    private SAMBinaryTagAndValue getAttributeMayBeNull(final short tag) {
```

Comment:
@SHuang-Broad Do we need this method at all?  I still don't really understand why we need to extract this out.

Reply:
is `getNullableAttribute` more consistent with Java naming schemes?

Reply:
it's not a naming issue anymore, it's just that if there's only 1 use of this, I don't see the need to extract it at all.